src/components/MyAccount/RequestsTab/CancelButton.tsx

```diff
@@ -77,6 +75,7 @@ const CancelButton = ({
           <Text sx={{ marginBottom: 0 }}>Cancel request?</Text>
         </Box>
       ),
+      //@ts-ignore
```

Comment:
Is this here because it's hard to determine the event type?

Reply:
no this is because we are overriding the expected type for this method, which is a method with no args. we are hacking the modal by availing ourselves of the event which is actually passed in to the callback. i'll add a comment explaining.

Reply:
Hopefully we make an update for this this sprint on the DS side.